ultra/ultra/utils/common.py

```diff
@@ -145,7 +144,7 @@ def copy_source_code(srcs, dst):
 
 
 def resize_im(im, size):
```

Comment:
What is the type for `size` here? This is the only location of `resize_im` in all of SMARTS